compiler/sfmt/dag.go

```diff
@@ -98,7 +98,7 @@ func (c *canonDAG) expr(e dag.Expr, parent string) {
 		c.write(" : ")
 		c.expr(e.Else, "")
 	case *dag.Call:
-		c.write("%s(", e.Name)
+		c.write("%s(", e.Name())
 		c.exprs(e.Args)
 		c.write(")")
```

Comment:
This output is wrong.

```console
$ super compile -C -dag 'op o a: (values a(this))  o lambda x: x+1'
null
| (
  values (this)
  | output main
)
```

Reply:
Hmm more test coverage.  Will work on this...

Reply:
Fixed it and added test